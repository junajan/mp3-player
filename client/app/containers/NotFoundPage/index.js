import React from 'react';
import Typography from '@material-ui/core/Typography';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    <Typography variant="h3" component="h2">
      Error 404
    </Typography>
    <br />
    <Typography variant="subtitle2">
      The requested page was not found ..
      <br />
      Here you can go to the <Link to="/">homepage</Link>.
    </Typography>
  </div>
);

export default NotFound;
