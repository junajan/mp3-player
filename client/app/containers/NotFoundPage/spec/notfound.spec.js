import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../index';

describe('Error404 page', () => {
  it('should display error message', () => {
    const page = shallow(<NotFoundPage />);
    expect(page).toMatchSnapshot();
  });
});
