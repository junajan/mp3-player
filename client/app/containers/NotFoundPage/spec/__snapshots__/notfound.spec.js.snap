// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 page should display error message 1`] = `
<div>
  <WithStyles(Typography)
    component="h2"
    variant="h3"
  >
    Error 404
  </WithStyles(Typography)>
  <br />
  <WithStyles(Typography)
    variant="subtitle2"
  >
    The requested page was not found ..
    <br />
    Here you can go to the 
    <Link
      replace={false}
      to="/"
    >
      homepage
    </Link>
    .
  </WithStyles(Typography)>
</div>
`;
